apis/v1alpha1/opentelemetrycollector_webhook.go

```diff
@@ -202,22 +203,12 @@ func (r *OpenTelemetryCollector) validateCRDSpec() error {
 			return fmt.Errorf("the OpenTelemetry Spec autoscale configuration is incorrect, minReplicas should be one or more")
 		}
 
-		if r.Spec.Autoscaler != nil && r.Spec.Autoscaler.Behavior != nil {
-			if r.Spec.Autoscaler.Behavior.ScaleDown != nil && r.Spec.Autoscaler.Behavior.ScaleDown.StabilizationWindowSeconds != nil &&
-				*r.Spec.Autoscaler.Behavior.ScaleDown.StabilizationWindowSeconds < int32(1) {
-				return fmt.Errorf("the OpenTelemetry Spec autoscale configuration is incorrect, scaleDown should be one or more")
+		if r.Spec.Autoscaler != nil {
+			err := checkAutoscalerSpec(r.Spec.Autoscaler)
+			if err != nil {
+				return err
 			}
```

Comment:
If you move this check to the end of the function (after the ingress check), we can just return `checkAutoscalerSpec(r.Spec.Autoscaler)` which either returns nil or err

Reply:
Using this suggestion now - thanks!